app/views/index.erb

```diff
@@ -0,0 +1,35 @@
+<body>
```

Comment:
Don't include `body` tags in your views when you already have `body` tags in your layout.